2996_reflection/reflection.md

```diff
@@ -6403,7 +6411,7 @@ consteval vector<info> members_of(info r);
 
 [#]{.pnum} A member `$M$` of a class or namespace is _members-of-reachable_ from a point `$P$` if there exists a declaration `$D$` of `$M$` that is reachable from `$P$`, and either `$M$` is not TU-local or `$D$` is declared in the translation unit containing `$P$`.
 
-[#]{.pnum} *Returns*: A `vector` containing reflections of all members-of-representable members of the entity represented by `r` that are members-of-reachable from some point in the evaluation context ([expr.const]).
+[#]{.pnum} *Returns*: If `r` represents a closure type, then a `vector` whose contents are unspecified. Otherwise, a `vector` containing reflections of all members-of-representable members of the entity represented by `r` that are members-of-reachable from some point in the evaluation context ([expr.const]).
```

Comment:
Is it really *completely* unspecified? Like, if I have
```cpp
[a=1, b=2]{ return a+b; }
```
I would certainly expect at the very least: `a`, `b`, an `operator()`, and a conversion function to `int(*)()`.

Reply:
I think we can get away with: Call operator, cast operator, and non-static data members corresponding to entities captured by copy.

Done. Also some light reformatting (added bullets and bullet numbers, fixed a typo); let me know if anything looks weird.